server/controllers/finance/reports/index.js

```diff
@@ -14,5 +14,5 @@ exports.invoices = require('./invoices');
 exports.vouchers = require('./vouchers');
 exports.cash     = require('./cash');
 exports.cashflow = require('./cashflow');
-exports.financialPatient = require('./financial.patient');
+exports.financialPatient = require('./financial.patient').report;
```

Comment:
I know you didn't write this, but `financialPatient` is a bit of a confusing name.  Maybe `patientFinancialStatus`?  Does it give the account number of the patient, or just return the balance, debit, credits to the patient?